### ISSUE TITLE
Unresolved resource dependencies [CodeBuildEvent]

### ISSUE THREAD

**Issue by maplion** [galaxy4public/aws-cloudformation-badges#4]
Using the Serverless Framework for deployment, I receive "An error occurred: Badges - Template format error: Unresolved resource dependencies [CodeBuildEvent] in the Resources block of the template" when deploying.

I feel like this should work, but I may try your aws cli option as a workaround.

**Comment by maplion** [comment]
Even with the following format and using the CLI instead

```shell
aws cloudformation deploy --template-file badges.template \
        --stack-name badges \
        --parameter-overrides \
                CodeBuildProjects=project1,project2,project3 \
                CodePipelines=pipeline1,pipeline2 \
        --capabilities CAPABILITY_IAM
```
I get similar results:  "An error occurred (ValidationError) when calling the CreateChangeSet operation: Template format error: Unresolved resource dependencies [CodeBuildEvent] in the Resources block of the template"

**Comment by maplion** [comment]
I was able to make this work by manually choosing a condition I wanted from the template and deleting the conditions out of template options.

**Comment by maplion** [comment]
Do you have a more readable, non-minified version of the Lambda code?  I could easily tranform this entire thing into a Serverless setup instead of using an AWS SAM Template -- I think it would be cleaner and easier to use and read.

**Comment by galaxy4public** [comment]
Re: more readable Lamba code - no, I don't have it since it was written by hand and it is small enough to be able to decipher it withing 5-10 minutes.

The other concern I addressed with PR #6
